lib/components/Pagination/index.tsx

```diff
@@ -13,11 +13,12 @@ import "./pagination.scss";
 export const Pagination = ({
     contentPerPage,
     contentLength,
+    initialPage,
```

Comment:
So just to understand , the initialPage number will only be set on first render? and then on 
`onClickPagination` . We are not changing the initialPage from the consumer side ? 
If we are changing , I think we should use a better name for `initialPage`

Reply:
initialPage will be set not only  on first render, but in case if it changed too. Thre is a useEffect below for setting it into state